bancor_research/scenario_generator/monte_carlo_generator.py

```diff
@@ -0,0 +1,616 @@
+from decimal import Decimal
+import pandas as pd
+from bancor_simulator.v3.spec import get_vault_balance
+
+from bancor_research.bancor_simulator.v3.spec import *
```

Comment:
Ideally, we can/should import only `BancorDapp` here, which would allow us to:
```
from bancor_research.bancor_simulator.v3.spec import BancorDapp as sBancorDapp
from bancor_research.bancor_emulator.v3.spec import BancorDapp as eBancorDapp
...
dapps = [
    BancorDapp(
        cooldown_time=cooldown_time,
        whitelisted_tokens=whitelisted_tokens,
        bnt_min_liquidity=bnt_min_liquidity,
        price_feeds=price_feed,
    )
    for BancorDapp in [sBancorDapp, eBancorDapp]
```
Then later run everything for each element in `dapps`.

Or something similar to that, anyway...

But generally speaking, relying only on the `BancorDapp` class as interface to the lower level (i.e., as "entry point" to the entire 'spec' module), would allow us to easily swap between the simulator and the emulator).